### PR TITLE
CASSANDRA-17130: Log missing peers in StartupClusterConnectivityChecker

### PR DESCRIPTION
**Pull request by yifan-c**

### CODE REVIEW

File: src/java/org/apache/cassandra/net/StartupClusterConnectivityChecker.java [hunk 1]
```diff
@@ -118,7 +121,7 @@ public boolean execute(Set<InetAddressAndPort> peers, Function<InetAddressAndPor
                         TimeUnit.NANOSECONDS.toSeconds(timeoutNanos));
         }
 
-        AckMap acks = new AckMap(3);
+        AckMap acks = new AckMap(3, peers);
```
Comment (spodkowinski) [inline src/java/org/apache/cassandra/net/StartupClusterConnectivityChecker.java:125]: Why would 3 be the correct value to use here? Not part of the change set, but can you please add a code comment?

**Review by spodkowinski — APPROVED** [review]

File: src/java/org/apache/cassandra/net/StartupClusterConnectivityChecker.java (hunk 1 above)
Comment (yifan-c) [reply src/java/org/apache/cassandra/net/StartupClusterConnectivityChecker.java:125]: Good idea. I will add the comment.

3 is here because the starting node need to get 3 acks, i.e. small message connection, large message connection and alive event from gossip, from each peer
